### PR TITLE
 [CodeClean] possible leak when inserting item into hashtable

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, handle error case when inserting new data into hashtable.

### CODE REVIEW

**Review by niley7464 — APPROVED** [review]
LGTM!

File: ext/nnstreamer/tensor_filter/vivante/build_run.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # @Author: Geunsik Lim <geunsik.lim@samsung.com>
-# @brief: The simple script is to build and run the VIM3/Vivante fiflter 
+# @brief: The simple script is to build and run the VIM3/Vivante fiflter
```
Comment (djeong20) [inline ext/nnstreamer/tensor_filter/vivante/build_run.sh:4]: typo?
```suggestion
# @brief: The simple script is to build and run the VIM3/Vivante filter
```

**Review by djeong20 — APPROVED** [review]

File: ext/nnstreamer/tensor_filter/vivante/build_run.sh (hunk 1 above)
Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_filter/vivante/build_run.sh]: thanks, updated.

**Review by anyj0527 — APPROVED** [review]
LGTM👍

File: gst/nnstreamer/tensor_filter/README.md [hunk 2]
```diff
@@ -3,13 +3,13 @@ title: tensor_filter
 ...
 
 # NNStreamer::tensor\_filter
-This is the main element of the whole NNStreamer project. This connects gstreamer data stream with neural network frameworks (NNFW) such as Tensorflow or Caffe.
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_filter/README.md:6]: The two spaces in the markdown file are for line separation.
Please keep this style.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/README.md]: updated, plz review the changes again.

**Pull request merged**